core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -59,6 +61,9 @@
 
     private String performanceCountersSection = DEFAULT_PERFORMANCE_MODULES_PACKAGE;
 
+    final static String ENV_VARIABLE_I_KEY = "APPLICATION_INSIGHTS_IKEY";
+    final static String JVM_VARIABLE_I_KEY = "-D" + ENV_VARIABLE_I_KEY;
```

Comment:
no, I started with that since I wanted to use one name so the user is more likely to remember, and easier docs/blogs

But, I don't have any objections to any other name, lets consult with Harel too, and then I will change that name